### PR TITLE
Use more gradle/gradle-build-action

### PR DESCRIPTION
**Pull request by Goooler**
Addressed from #209.

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -22,8 +22,9 @@ jobs:
       with:
         java-version: 11
         distribution: 'zulu'
+    - uses: gradle/gradle-build-action@v2
     - name: Run sanity check
-      run: ./gradlew --scan --stacktrace test
+      run: ./gradlew --stacktrace test
```
Comment (szpak) [inline .github/workflows/e2e.yml:27]: Why did you remove the build scan? It is useful (among others) to get the detailed test failure from the CI build?

Comment (Goooler) [reply .github/workflows/e2e.yml:27]: Ooops, I missed there is no `publishAlways` in `buildScan`.

Comment (Goooler) [reply .github/workflows/e2e.yml:27]: 3f2505745c3806e2e8f0f31f82410775a87b22c5

**Comment by szpak** [comment]
Thanks.

**Pull request merged**